Readme.md

```diff
@@ -20,7 +20,7 @@ Add the following code to it:
 
 ```js
 module.exports = {
-  addons: ['@etchteam/storybook-addon-css-variables-theme/dist/preset'],
+  addons: ['@etchteam/storybook-addon-css-variables-theme'],
```

Comment:
Will the `/dist/preset` path continue to work, or will it break? If it'll break, then we should be doing a semver major bump, no?